apps/web/cypress/tests/integration-store.spec.ts

```diff
@@ -1,6 +1,14 @@
+// @ts-nocheck
 import { ChannelTypeEnum, InAppProviderIdEnum } from '@novu/shared';
 
-describe.skip('Integration store page', function () {
+Cypress.on('window:before:load', (win) => {
+  win._cypress = {
+    ...win._cypress,
+    IS_INTEGRATIONS_LIST_PAGE_ENABLED: 'false',
+  };
+});
```

Comment:
Set the `_cypress.IS_INTEGRATIONS_LIST_PAGE_ENABLED` flag to false, it is then used in the configuration file where we do set up the env variables. 
I wasn't able to use the `window._env_` property as it overrides somewhere during the application initialization all the values.